cmd/client-fs.go

```diff
@@ -765,34 +699,8 @@ func (f *fsClient) listInRoutine(contentCh chan<- *clientContent, isMetadata boo
 			if fi.Mode()&os.ModeSymlink == os.ModeSymlink {
 				fp := filepath.Join(fpath, fi.Name())
 				fi, e = os.Stat(fp)
-				if os.IsPermission(e) {
-					contentCh <- &clientContent{
-						Err: probe.NewError(PathInsufficientPermission{Path: pathURL.Path}),
-					}
-					continue
-				}
-				if os.IsNotExist(e) {
-					// Lstat makes no attempt to follow the broken link.
-					_, e = os.Lstat(fp)
-					contentCh <- &clientContent{
-						URL:  pathURL,
-						Size: -1,
```

Comment:
@harshavardhana Can you take a look at this comment here - https://github.com/minio/mc/pull/2607#issuecomment-445264783. The size was set to -1 because of this purpose of file tagging. To identify infected files and color code them. Do you think it is not needed?

Reply:
We don't actually do any of that, we used it as a way to log some warnings but its not useful in any manner. Every log should have a purpose where the log becomes actionable to the user. 

Our logging is kind of non-actionable we don't need to preserve symlinks because if they can be followed so be it we will copy, if we cannot we just ignore and continue. 

Preserving broken links is not supported so logging that is not useful.

Reply:
Ok. Agreed :+1: